test/acceptance/file-utils.spec.js

```diff
@@ -36,46 +36,68 @@ describe('lookupFiles', function () {
     }
   });
 
-  (symlinkSupported ? it : it.skip)('should not choke on symlinks', function () {
-    expect(utils.lookupFiles(tmpDir, ['js'], false))
-      .to
-      .contain(tmpFile('mocha-utils-link.js'))
-      .and
-      .contain(tmpFile('mocha-utils.js'))
-      .and
-      .have
-      .length(2);
-    expect(existsSync(tmpFile('mocha-utils-link.js')))
-      .to
-      .be(true);
-    fs.renameSync(tmpFile('mocha-utils.js'), tmpFile('bob'));
-    expect(existsSync(tmpFile('mocha-utils-link.js')))
-      .to
-      .be(false);
-    expect(utils.lookupFiles(tmpDir, ['js'], false))
-      .to
-      .eql([]);
+  describe('.lookupFiles', function () {
+    (symlinkSupported ? it : it.skip)('lookupFiles keke should not choke on symlinks', function () {
```

Comment:
Test description should omit "lookupFiles" (which is in the suite name/description, which is generally printed before the test name) and "keke".